packages/flutter/lib/src/cupertino/page.dart

```diff
@@ -424,19 +424,12 @@ class _CupertinoEdgeShadowDecoration extends Decoration {
   }
 
   @override
-  DiagnosticsNode toDiagnosticsNode({
-    String name,
-    DiagnosticsTreeStyle style: DiagnosticsTreeStyle.singleLine,
-  }) {
-    return new DiagnosticsNode.lazy(
-      name: name,
-      value: this,
-      style: style,
-      description: '$runtimeType',
-      fillProperties: (List<DiagnosticsNode> properties) {
-        properties.add(new DiagnosticsProperty<LinearGradient>('edgeGradient', edgeGradient));
-      },
-    );
+  String toStringShort() => '$runtimeType';
```

Comment:
I don't currently have the decoration base class override toStringShort. I could do that if it is believed that no decoration subclasses should use the default className#hashCode toStringShort

One theory is it is more idiomatic to specify that the short string is $runtimeType only on the class that implements operator ==, etc.